flybywiresim/a32nx
# feat: ISIS brightness buttons push&hold #4736

Fixes #4184 #4736 #4794 

## Summary of Changes
Brightness buttons can now be held down to increase/decrease brightness (push and hold). As per issue, time taken to increase brightness to full from minimum is roughly 4 seconds.

## Screenshots (if necessary)


![aBr61h3TRS](https://user-images.githubusercontent.com/15316958/117405821-cbfb2880-af3e-11eb-9362-8bcff957863b.gif)

## References

See PR #4736 #4794 
See Issue #4184 

https://discord.com/channels/738864299392630914/758126266859388948/832179131152203806




## Additional context



Discord username (if different from GitHub): 2Cas#1022

## Testing instructions


1. In the self-test and altimeter views, use standby instrument's brightness push buttons to adjust the brightness of the display. 
2. Pressing and holding as well as single sharp presses should both work as expected.
3. Press the BUGS button. In this mode, the standby instrument's brightness push buttons should move the cursor that selects the heading/altitude bugs that are being adjusted by the altimeter knob. The brightness should not change. 
4. Use the integrated panel emissive knob to adjust the brightness of the panel lighting. The buttons should light up or stop emitting light as expected.
5. Attempt to break the brightness control by clicking on both buttons randomly. The brightness adjustment should continue working.


## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, click on the bottom **PR** tab
1. Click on the **A32NX** download link at the bottom of the page
